packages/textarea/src/stories/tw-ui-textarea-theme.config.ts

```diff
@@ -0,0 +1,14 @@
+import { TailwindVariants } from '@unicodernsui/custom-theme'
```

Comment:
you can remove this file, since tailwind stories cannot be defined until Lucia defines how the textarea will look